packages/jellyfish-api-core/src/category/account.ts

```diff
@@ -294,10 +312,18 @@ export interface AccountHistoryOptions {
 }
 
 export interface UtxosToAccountPayload {
-  [key: string]: string
+  [key: string]: TokenType
+}
+
+export interface AccountToAccountPayload {
+  [key: string]: TokenType
```

Comment:
duplicated... you can name it properly `toAccountPayload`

Reply:
The problem if rename this as toAccountPayload, reading utxosAToAccountPayload and toAccountPayload seems weird

Reply:
i mean its duplicated.. combine them one and named `toAccountPayload`

Reply:
Hmm, we will have AccountToUtxosPayload in my next accounttoutxos PR.  So these are the variables:- 

utxostoaccountpayload
accounttoaccountpayload
accounttoutxospayload

toAccountPayload seems good for 1st 2. But not for the 3rd.

Maybe after consider 3rd, you can think of a good name for payload. Yeah payload the word itself is too general

Reply:
properly `BalanceTransferPayload`